src/explore-education-statistics-admin/src/services/releaseDataFileService.ts

```diff
@@ -197,6 +202,9 @@ const releaseDataFileService = {
       })
       .then(response => downloadFile(response, fileName));
   },
+  cancelImport(releaseId: string, fileId: string): Promise<void> {
+    return client.post(`/release/${releaseId}/data/${fileId}/import/cancel`);
```

Comment:
I noticed this endpoint ended up in `BauFileImportsController` which is basically the last place I would have guessed it would be! Would suggest moving the endpoint into `ReleasesController` with the other methods data endpoint, or split out a `ReleaseDataController` with all of those endpoints grouped together.

Somewhat arbitrary splitting up of code based on its 'BAUness' is a bit of an anti-pattern and really will only lead to confusion about where things should be placed. Would advise thinking about things more RESTfully in terms of resources/domains.

Reply:
Yeah that's a good point.  When I first came to this piece of work, it was all about a new BAU screen for seeing all of the ongoing file imports, so I imagined it eventually going in there, but was very much in a BAU mindset.

I will moved it to ReleasesController as you suggest.